salt/utils/schema.py

```diff
@@ -687,6 +688,15 @@ def as_requirements_item(cls):
     #    raise NotImplementedError
 
 
+class EventSchema(Schema):
+
+    tag = None
+
+    @classmethod
+    def match(cls, tag):
```

Comment:
by default we use regex with tag matches but can be overwritten for tags where the minion id is present in the tag. 
Also currently this regex match is way to open, I'll update it to be stricter if you all agree.